cmd/bee/cmd/start.go

```diff
@@ -320,7 +321,14 @@ func (c *command) configureSigner(cmd *cobra.Command, logger logging.Logger) (co
 			return nil, err
 		}
 
-		signer, err = clef.NewSigner(externalSigner, clefRPC, crypto.Recover)
+		wantedAddress := c.config.GetString(optionNameClefSignerAddress)
+		var overlayEthAddress *common.Address = nil
+		if wantedAddress != "" {
+			ethAddress := common.HexToAddress(wantedAddress)
+			overlayEthAddress = &ethAddress
+		}
+
+		signer, err = clef.NewSigner(externalSigner, clefRPC, crypto.Recover, overlayEthAddress)
```

Comment:
What happens when there is no previously-existing account in clef? When/where do we make this?

Reply:
account management is done by the user on the clef side (or in our case by the `bee-clef` package). bee expects the account to already be present.